data/scrape/lib/unscraped.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Please run `git checkout -- unscraped.json` (in this directory, or use the full path from another directory). This doesn't need checked in.